test/index.test.js

```diff
@@ -1,94 +1,111 @@
 const { test } = require('tap')
 const Fastify = require('fastify')
 const fastifySlonik = require('../index')
-const { BAD_DB_NAME, connectionString, connectionStringBadDbName } = require('./helpers')
 
-test('Namespace should exist:', (t) => {
-  t.test('fastify.slonik', t => {
-    t.plan(4)
+const connectionString = process.env.DATABASE_URL
+const BAD_DB_NAME = 'db_that_does_not_exist'
+const connectionStringBadDbName = connectionString.replace(/\/[^/]+$/, '/' + BAD_DB_NAME)
 
-    const fastify = Fastify()
-    t.teardown(() => fastify.close())
-
-    fastify.register(fastifySlonik, {
-      connectionString
-    })
-    fastify.ready((err) => {
-      t.error(err)
-
-      t.ok(fastify.slonik)
-      t.ok(fastify.slonik.pool)
-      t.ok(fastify.slonik.connect)
-    })
-  })
-  test('fastify.sql', (t) => {
-    t.plan(2)
+test('Namespace should exist:', async t => {
+  const fastify = Fastify()
 
-    const fastify = Fastify()
-    t.teardown(() => fastify.close())
+  t.teardown(() => fastify.close())
 
-    fastify.register(fastifySlonik, {
-      connectionString
-    })
+  fastify.register(fastifySlonik, { connectionString })
+  await fastify.ready()
 
-    fastify.ready((err) => {
-      t.error(err)
+  t.test('fastify.slonik', async t => {
+    t.ok(fastify.slonik)
+    t.ok(fastify.slonik.pool)
+    t.ok(fastify.slonik.connect)
+    t.ok(fastify.slonik.query)
+    t.ok(fastify.slonik.transaction)
+    t.ok(fastify.slonik.exists)
+  })
 
-      t.ok(fastify.sql)
-    })
+  t.test('fastify.sql', async t => {
+    t.ok(fastify.sql)
   })
```

Comment:
There isn't really a need nest these assertions in a `t.test`. Could we have them live in their parent scope? Also, for asserting the top-level decorator could we use `fastify.hasDecorator`?
```js
  t.ok(fastify.hasDecorator('slonik'), 'has slonik decorator')
  // ...
  t.ok(fastify.hasDecorator('sql'), 'has sql decorator')
```